pkgs/applications/networking/browsers/chromium/upstream-info.nix

```diff
@@ -6,9 +6,9 @@
     version = "73.0.3683.75";
   };
   dev = {
-    sha256 = "19fwzxnsd1parqghv4b2mif3cj1k1m5hzqnjsnglkgv6xnqny98g";
-    sha256bin64 = "1aag76dgcnwga1q4jdgm3ziqvxs1qvrsyqfd7bbsi7axz6qi135v";
-    version = "74.0.3724.8";
+    sha256 = "0kwpy0d3zcb6cf2g32sm6rgw2np6jdgq7d4jxjvm58xv1g9xb49c";
+    sha256bin64 = "1qb3mls2ln5pzg83yalvmwzjzzi058w7zxna7l2gzvnbrd9m97w7";
+    version = "74.0.3729.6";
```

Comment:
Is this version update necessary, to make the rest of the PR work?

Reply:
Very unlikely. I can revert and test right now if it's better to not update this for some reason?

Reply:
In the context of a PR it's better not to mix in incidental stuff. It's all about the review process. A reviewer (amongst other things) will read through the diff and ask themselves the same question for every hunk: "What's the purpose of this change?"

When I read a version update mixed in with a feature implementation, my answer to that question will be: "The feature must depend on that version". This didn't make any sense for this version update, so I asked ..